src/goto-symex/goto_symex.h

```diff
@@ -262,6 +262,8 @@ class goto_symext
   void symex_assign_rec(statet &state, const code_assignt &code);
   virtual void symex_assign(statet &state, const code_assignt &code);
 
+  void havoc_rec(statet &, const guardt &, const exprt &);
```

Comment:
It would be nice to add comments about the purpose of the method and its input parameters.